src/bvh.jl

```diff
@@ -1,7 +1,33 @@
 export BoundingVolumeHierarchy
 
+"""
+    AccelerationStructure
+
+Base Type for all Acceleration Structures. These can be used to speed up
+rendering by a significant amount. The main design behind an acceleration
+structure should be such that it can be used as a simple replacement for
+a vector of [`Object`](@ref)s.
+"""                  
 abstract type AccelerationStructure end
```

Comment:
I'm just wondering if this type is really needed (i.e., would adding methods that have a specific implementation for all acceleration structures be relevant?). It might still be useful if you plan on drawing the type hierarchy, for instance, so I'm not proposing to remove it (unless you think it makes things clearer).

Reply:
> would adding methods that have a specific implementation for all acceleration structures be relevant?

No, we can't have a specific implementation relevant to all acc. structures

This type is simply so that we have a clean type hierarchy.